### DISCUSSION TITLE
Alternatives, rewrites, etc

### DISCUSSION THREAD

**Discussion by explosion-mental** [AdnanHodzic/auto-cpufreq#506] [General]
Simply listing projects with similar goals and logic, [the blog post](https://foolcontrol.org/?p=3124) is a nice reference as to what to address, that I was able to find (and write :p):

- [auto-clock-speed](https://github.com/JakeRoggenbuck/auto-clock-speed) 
- [yablo - Yet Another Battery Linux Optimizer](https://github.com/sebastian-xyz/yablo)
- [racf - Rusty Auto CPU Frequencer](https://github.com/explosion-mental/racf) 
- [sacf - Simple Auto CPU Frequencer](https://github.com/explosion-mental/sacf) (uses config.h)

**Comment by eylles** [comment]
may as well mention my own tool [afreq.sh](https://github.com/eylles/afreq.sh), the most recent release tag has the option to run hooks for adding extra tweaks depending on the ac state, cpu stage and current governor, all while written in pure posix shell.
